static/js/components/PersonalTab_test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { assert } from 'chai';
+import R from 'ramda';
+
+import PersonalTab from './PersonalTab';
+import { PROGRAM_ENROLLMENTS } from '../constants';
+
+describe("PersonalTab", () => {
+
+  let renderPersonalTab = (props = {}) => {
```

Comment:
seems like you should just have this function accept a `selectedProgram` and set that value appropriately in the `PersonalTab` props. with the current implementation you need to repeat the structure of the props in the last test case. doesn't seem necessary